frontend/bank-transfer-ui/src/app/main/transfer-list/transfer-list.component.html

```diff
@@ -1,10 +1,96 @@
-
-<div class="content">
-
+<div class="wrapper">
   <div class="container">
```

Comment:
Znowu wrzucasz tą samą paczkę, tj. div z klasą wrapper, potem div z klasą container, potem div z klasą content. Przenieś to do parenta, żeby było globalne, albo zrób sobie jakiś komponent, który Ci zrobi to za Ciebie